src/test/compatibility/custom-json-formatter.js

```diff
@@ -1,14 +1,30 @@
 const { JsonFormatter } = require('cucumber')
+const { notIsNilOrEmpty } = require('../../web/routes/application/common/predicates')
 
 require('dotenv').config()
 
+function isMobileDevice () {
+  return !!process.env.BROWSER_STACK_DEVICE
+}
+
+function getBrowserVersion () {
```

Comment:
@gethyn1 there is almost certainly a nicer version of this, any good suggestions?

Reply:
I guess its a bit confusing that `getBrowserVersion()` is mixed with `isMobileDevice()`. I would maybe split them up where you call `getBrowserVersion()`:
```
...
'version': isMobileDevice() ? '' : getBrowserVersion()
...
```
.. but I'm not sure if that gives you the result you need? If not then I think this is fine.